src/App.js

```diff
@@ -73,7 +72,6 @@ function App() {
     <div className="App">
       { stateLogged === TRUE ? <SuspenseComponent lazyComponent={<Menu />} /> : <></> }
       <div className="AppMainContainer">
-      { stateLogged === NOTKNOWN ? <SuspenseComponent lazyComponent={<WhileLogging></WhileLogging>} /> : 
           <Routes>
```

Comment:
let's do something here to avoid the user waiting time without UI changes to notice

Reply:
while the user is loggin in correctly, I mean